code-challenges.js

```diff
@@ -22,47 +22,105 @@ const fruit4 = "kiwi"
 
 // Pseudo code:
 
+    // Make an arrow function that takes a number (temperature) and verifies if the number is below, at, or above another number (boiling point) by checking if the number is higher, lower, or equal  (  > , < , and ===  ) 
+    // Returns a string:
+            // "(Temperature) is below boiling point."
+            // "(Temperature) is at boiling point."
+            // "(Temperature) is above boiling point."
+    // The number that signifies boiling point is 212.
```

Comment:
Great pseudocode and awesome approach to this problem! Love that you used the catch all to account for not getting the input you need. One thing you can do is mention using string interpolation or that you'll set 212 to a variable just to be more descriptive.